### ISSUE TITLE
Address build warnings

### ISSUE THREAD

**Issue by aspnet-hello** [aspnet/Home#2511]
_From @mikeharder on Monday, May 8, 2017 12:34:14 PM_

Running `build.cmd` in Preview1 generates 9 build warnings:

```
Program.cs(25,13): warning CS0618: 'ConsoleLoggerExtensions.AddConsole(ILoggerFactory)' is obsolete: 'This method is obsolete and will be removed in a future version. The recommended alternative is to call the Microsoft.Extensions.Logging.AddConsole() extension method on the Microsoft.Extensions.Logging.LoggerFactory instance.' [D:\Git\DataProtection\samples\CustomEncryptorSample\CustomEncryptorSample.csproj]
Program.cs(27,13): warning CS0618: 'ConsoleLoggerExtensions.AddConsole(ILoggerFactory, LogLevel)' is obsolete: 'This method is obsolete and will be removed in a future version. The recommended alternative is to call the Microsoft.Extensions.Logging.AddConsole() extension method on the Microsoft.Extensions.Logging.LoggerFactory instance.' [D:\Git\DataProtection\samples\Redis\Redis.csproj]
Program.cs(27,13): warning CS0618: 'ConsoleLoggerExtensions.AddConsole(ILoggerFactory, LogLevel)' is obsolete: 'This method is obsolete and will be removed in a future version. The recommended alternative is to call the Microsoft.Extensions.Logging.AddConsole() extension method on the Microsoft.Extensions.Logging.LoggerFactory instance.' [D:\Git\DataProtection\samples\Redis\Redis.csproj]
Program.cs(25,13): warning CS0618: 'ConsoleLoggerExtensions.AddConsole(ILoggerFactory)' is obsolete: 'This method is obsolete and will be removed in a future version. The recommended alternative is to call the Microsoft.Extensions.Logging.AddConsole() extension method on the Microsoft.Extensions.Logging.LoggerFactory instance.' [D:\Git\DataProtection\samples\CustomEncryptorSample\CustomEncryptorSample.csproj]
Program.cs(35,13): warning CS0618: 'ConsoleLoggerExtensions.AddConsole(ILoggerFactory, LogLevel)' is obsolete: 'This method is obsolete and will be removed in a future version. The recommended alternative is to call the Microsoft.Extensions.Logging.AddConsole() extension method on the Microsoft.Extensions.Logging.LoggerFactory instance.' [D:\Git\DataProtection\samples\AzureBlob\AzureBlob.csproj]
C:\Users\mharder\.nuget\packages\internal.aspnetcore.sdk\2.1.0-rc2-15256\build\ApiCheck.targets(27,5): warning : No baseline file for net46 found in Microsoft.AspNetCore.DataProtection.Redis. [D:\Git\DataProtection\src\Microsoft.AspNetCore.DataProtection.Redis\Microsoft.AspNetCore.DataProtection.Redis.csproj]
C:\Users\mharder\.nuget\packages\internal.aspnetcore.sdk\2.1.0-rc2-15256\build\ApiCheck.targets(27,5): warning : No baseline file for net46 found in Microsoft.AspNetCore.DataProtection.AzureStorage. [D:\Git\DataProtection\src\Microsoft.AspNetCore.DataProtection.AzureStorage\Microsoft.AspNetCore.DataProtection.AzureStorage.csproj]
C:\Users\mharder\.nuget\packages\internal.aspnetcore.sdk\2.1.0-rc2-15256\build\ApiCheck.targets(27,5): warning : No baseline file for netstandard1.5 found in Microsoft.AspNetCore.DataProtection.AzureStorage. [D:\Git\DataProtection\src\Microsoft.AspNetCore.DataProtection.AzureStorage\Microsoft.AspNetCore.DataProtection.AzureStorage.csproj]
C:\Users\mharder\.nuget\packages\internal.aspnetcore.sdk\2.1.0-rc2-15256\build\ApiCheck.targets(27,5): warning : No baseline file for netstandard1.5 found in Microsoft.AspNetCore.DataProtection.Redis. [D:\Git\DataProtection\src\Microsoft.AspNetCore.DataProtection.Redis\Microsoft.AspNetCore.DataProtection.Redis.csproj]
```

_Copied from original issue: aspnet/DataProtection#232_

**Comment by aspnet-hello** [comment]
_From @natemcmaster on Friday, June 2, 2017 12:52:47 PM_

The logger API's were fixed in #234. We still need to add baseline files

**Comment by natemcmaster** [comment]
This was already fixed by Ajay in https://github.com/aspnet/DataProtection/commit/ac3cbb5f8d1e73784c0d28129ea45ecc789e4f28
